js/core/include-config.js

```diff
@@ -21,7 +21,7 @@ define(
           }
           return val;
         }
-        script.innerText = JSON.stringify(conf, confFilter, 2);
+        script.innerHTML = JSON.stringify(conf.initialUserConfig, confFilter, 2);
```

Comment:
I don't understand why this wouldn't work, but frontend dev is full of mysteries.